doc/read-the-docs-site/adr/index.rst

```diff
@@ -30,5 +30,4 @@ The general process for creating an ADR is:
    :maxdepth: 1
    :titlesonly:
 
-   0001-record-architecture-decisions
-   0002-steppable-cek-machine
```

Comment:
I will bring this back once https://github.com/input-output-hk/plutus/pull/4930#issuecomment-1301769289 has been settled